tests/test_job_management.py

```diff
@@ -1123,6 +1134,55 @@ def test_email_very_long_request_url(dispatcher_long_living_fixture, dispatcher_
 This will be fixed in a future release.""" in email_data
 
 
+def test_email_scws_list(dispatcher_live_fixture,
```

Comment:
oh, could we use this test to make clear our (argumably, somewhat unnecessarily convoluted) logic, in all of the following cases:

* `use_scws` NOT set and `scw_list` set (API and scw list result)
* `use_scws` NOT set and `scw_list` NOT set (API and time range result)
* `use_scws` in `form_list`, `user_file` and `scw_list` set (frontend, scw list result) Not sure about this one.
* `use_scws`==`no` (frontend, time range result)
* `use_scws`!=`no` and `scw_list` NOT set (frontend, exception)

Sorry to stress on this one, but it's a thing which unfortunately caused confusion in the past as well as at this late development stage.

Net result in all cases should be that products URL contains `use_scws` set correspondingly to the product kind, and API code never contains `use_scws` but contains `scw_list` ONLY if scw_list is used.

ALTERNATIVELY, better, would to make API behave exactly like frontend. This would introduce a breaking change in oda_api, meaning that new dispatcher would be only allowed with new oda_api. We could do this, if we make dispatcher refuse older API versions.
Maybe it's too much organization to do this. We can discuss tomorrow too.

I would suggest, for now, to complete the test.